### ISSUE TITLE
Can't control gateway subdevice by using miiocli

### ISSUE THREAD

**Issue by ssp2** [rytilahti/python-miio#995]
I can't use miiocli (ver.0.5.4) to control subdevice of gateway 2 (with developer mode opened). My subdevice is 2 channel aqara relay
(lumi.relay.c2acn01)

I tried all possible options with raw_command. For example:
miiocli --ip 192.168.1.xx --token tokentoken raw_command get_prop  '["lumi.158d0003f45775","load_power"]'

And hundreds of other variants. But the result is "error: method unknown" or error in json, or "None".

I got tired in experimenting and had to write my own python script similar to miiocli to control my relay, where everything functions well. I can get states of channels and change this states. I am ready to lay out this script on github, but I think, that it is better to improve the functionality of miiocli to support subdevices of the gateway. Or maybe I simply did not find the proper usage of miiocli, then somebody, show the correct way of its usage.

**Comment by starkillerOG** [comment]
@ssp2 subdevices can be controlled using very simple python scripts using python-miio.
However the subdevices can indeed not be controlled using command line.

I think the reason is because you need multiple sequential operations to control subdevices (first the gateway needs to find and report all subdevices so the python classes can be made for the appropriate subdevices) only then a subdevice can be controlled though that class. I am not sure if such a thing (sequential commands) is possible in command line (compared to just running a single standalone command).

**Comment by ssp2** [comment]
> @ssp2 subdevices can be controlled using very simple python scripts using python-miio.

You are right. Scrip is easy, but to write it is difficult, because there is no any documentation. Here is my realization https://github.com/ssp2/xiaomi-relay2

**Comment by starkillerOG** [comment]
I just made a PR to improve your script a bit: https://github.com/ssp2/xiaomi-relay2/pull/1
I would recommand using the build in functions for the switch and not make them again, in this way the script will not only work for this specific relay but for all xiaomi-miio switches out there.

**Comment by ssp2** [comment]
> I just made a PR to improve your script 

Thanks, I will check the functionality and aprove your commit later.
